indico/modules/events/registration/templates/management/import_registrations.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout/base.html' %}
+{% from 'forms/_form.html' import form_header, form_rows, form_footer  %}
+{% from 'message_box.html' import message_box %}
+
+{% block content %}
+    {% call message_box('highlight', large_icon=true) %}
+        {% trans num_columns=6 -%}
+            You should upload a CSV (comma-separated values) file with exactly {{num_columns}} columns in the
+            following order:
+            <ul>
+                <li>First Name</li>
+                <li>Last Name</li>
+                <li>Affiliation</li>
+                <li>Position</li>
+                <li>Phone Number</li>
+                <li>E-mail</li>
```

Comment:
Wouldn't it make more sense to put First/Last/Email first since those are probably the most common fields?

Reply:
I put 'e-mail' at the end because some of the other fields may be empty (e.g. position), and the resulting file may end up having < 6 columns.